components/contribution-flow/index.js

```diff
@@ -872,6 +906,50 @@ class CreateOrderPage extends React.Component {
               customData={customData}
               onCustomFieldsChange={this.handleCustomFieldsChange}
             />
+            <FeesOnTopContainer p={3} mt={3}>
+              <Box maxWidth="75%">
+                <P fontSize="Caption" my={2}>
+                  <FormattedMessage
+                    defaultMessage="Open Collective has waived 100% of our fees for COVID-19 Relief Collectives. We rely on the generosity
```

Comment:
It's already a detail, sorry, but beware of the line break within the string, remove it.

Reply:
With &nbsp; between the sentences you mean?

Reply:
Nope, there should be no line break in the `defaultMessage`, will propose the change.